CHANGELOG.md

```diff
@@ -1,5 +1,208 @@
 # Changelog
 
+## 5.0.0.rc1
+
+This is a major Teleport release with a focus on new features, functionality, and
+bug fixes. It’s a substantial release and users can review [5.0 closed issues](https://github.com/gravitational/teleport/milestone/39?closed=1)
+on Github for details of all items.
+
+#### New Features
+Teleport 5.0 is a major version and an introduction to Teleport's Unified Access
+Plane. Going beyond just SSH access, with first class Kubernetes support
+and the new addition of Application Support.
+
+##### Teleport Application Access
+Teleport can be used to provide secure access to Applications. We've
+built this Teleport Application Access with an eye for securing those internal apps
+which might once have lived on the VPN or have a simple OAuth mechanism but with little
+to no audit trail.  We've tested everything from router control panels to single
+page javascript apps.
+
+Adding an Application follows the same UX as adding servers. An invite token needs
+to be created. This can be static or dynamic.
+
+```bash
+$ tctl tokens add --type=app
+```
+
+A Teleport node can be used to route to Application from the unified
+access plane. The Teleport Application Access service can be deployed alongside applications, enabling teams
+to setup access on a loopback address so only Teleport Application Access can access the app.
+
+```yaml
+# ...
+proxy_service:
+  # We've extended support for https certs. Teleport can now load multiple
+  # TLS certificates. In the below example we've obtained a wildcard cert
+  # that'll be used for proxying the applications.
+  https_keypairs:
+  - key_file: /etc/letsencrypt/live/teleport.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/teleport.example.com/fullchain.pem
+  - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
+# ...
+app_service:
+   # Teleport Application Access is enabled.
+   enabled: yes
+   # We've a sample debugger app that'll check
+   # that Teleport Application Access is working
+   # and will output JWT tokens.
+   # https://dumper.teleport.example.com:3080/
+   debug_app: true
+   apps:
+   # Teleport Application Access can be used to proxy any HTTP Endpoint
+   # Note: Name can't include any spaces
+   - name: "internal-dashboard"
+     uri: "http://10.0.1.27:8000"
+     # By default Teleport will make this application
+     # available on sub-domain. Thus the importance of setting up
+     # wilcard DNS. If you want, it's possible to setup a vanity
+     # url. DNS records should point to the proxy server.
+     # internal-dashboard.teleport.example.com
+     # Example Vanity URL for the internal-dashboard app.
+     # public_addr: "dashboard.acme.com"
```

Comment:
This code comment can be shortened. IMO, we don't need to explain how to configure certain behavior, just that it can be done. 

```suggestion
     # By default Teleport makes this application
     # available on a sub-domain like teleport.example.com.
     # You can point public addresses DNS records to the application access proxy server
     # and have redirect users from dashboard.example.com to internal-dashboard.teleport.example.com
```